app/db.py

```diff
@@ -1,11 +1,25 @@
 from flask_pymongo import PyMongo
+from flask_mail import Mail
 
 
 def init_db():
     mongo = PyMongo()
     return mongo
 
 
-def get_db(app, mongo):
+def init_mail():
+    mail = Mail()
+    return mail
+
+
+def get_db(app, mongo, mail):
     app.config["MONGO_URI"] = "mongodb://localhost:27017/my_db"
+
+    app.config['MAIL_SERVER'] = 'smtp.gmail.com'
```

Comment:
why we are doing the email configs in the get_db function? That represent the db configs only